app/src/userpages/components/ProfilePage/ProfileSettings/index.jsx

```diff
@@ -34,7 +37,7 @@ type Props = StateProps & DispatchProps
 
 const options = moment.tz.names().map((tz) => ({
     value: tz,
-    label: tz,
+    label: `(UTC${moment.tz(tz).format('Z')}) ${tz}`.replace(/\//g, ', ').replace(/_/g, ' '),
```

Comment:
Not sure if this would be clearer:
```suggestion
    label: `${moment.tz(tz).format('(UTCZ)')} ${tz.replace('/', ', ')}`,
```
You can decide

Reply:
Left mine in, need to replace also all occurrences of underscores (eg. "Isle_of_Man" -> "Isle of Man")